### ISSUE TITLE
Running jake local exits

### ISSUE THREAD

**Issue by rjlee** [Microsoft/TouchDevelop#32]
On Mac OS X 10.9.5, node v0.12.0:

Running `jake local` gives:
````
host:TouchDevelop me$ jake local
[P] modifying in-place all css files
host:TouchDevelop me$
````

Amending Jakefile seemed to fix this issue:

````
diff --git a/Jakefile b/Jakefile
index 6cba6ae..fdec998 100644
--- a/Jakefile
+++ b/Jakefile
@@ -352,7 +352,7 @@ Object.keys(concatMap).forEach(function (f) {
     });
 });

-task('log', [], { async: true }, function () {
+task('log', [], { async: false }, function () {
   if (process.env.TRAVIS) {
     console.log("[I] dumping the last few commits in build/gitlog.txt");
     runAndComplete([ "git log -n 100 --pretty='format:%H%x20%ci%x20%cN%n%w(90,4,4)%s%n%w(90,4,4)%b' > build/gitlog.txt" ], this);
````

**Comment by pelikhan** [comment]
Same issue on windows. Settings 'async: false' works because git seems to take ages to fetch the log. There's an easier way to solver this:

- use the ``TRAVIS_COMMIT_RANGE`` environment variable to get the commit range
- use the URL described here to redirect the user to a nice view that shows the editor https://github.com/blog/609-tracking-deploys-with-compare-view

**Comment by pelikhan** [comment]
I'm fixing this.

**Comment by pelikhan** [comment]
fixed a04ebe0025f5945a4df6d54c441db7ccbf500d6e
